src/ui/ui_settings/public/send_request.js

```diff
@@ -1,20 +1,25 @@
 import chrome from 'ui/chrome';
 
 export async function sendRequest({ method, path, body }) {
-  const response = await fetch(chrome.addBasePath(path), {
-    method,
-    body: JSON.stringify(body),
-    headers: {
-      accept: 'application/json',
-      'content-type': 'application/json',
-      'kbn-xsrf': 'kibana',
-    },
-    credentials: 'same-origin'
-  });
+  chrome.loadingCount.increment();
+  try {
+    const response = await fetch(chrome.addBasePath(path), {
+      method,
+      body: JSON.stringify(body),
+      headers: {
+        accept: 'application/json',
+        'content-type': 'application/json',
+        'kbn-xsrf': 'kibana',
+      },
```

Comment:
You don't need to fix this as part of this PR since it's a bug in the original implementation, but this should really be sending the `kbn-version` as well, so the whole "your browser is out of date" logic works properly for these requests as well.

Reply:
Send `kbn-version` instead of `kbn-xsrf`?

Reply:
https://github.com/elastic/kibana/pull/18746